src/components/menuPage/MenuList.js

```diff
@@ -0,0 +1,30 @@
+import { useState,useEffect} from 'react';
+
+import ProductList from './ProductList';
+import SearchBar from './SearchBar';
+import FilterTagBar from './FilterTagBar';
+
+
+
+const MenuList = () => {
+    let [foodData,setFoodData] = useState()
+
+    useEffect(() => {
+        const getAllProducts = async ()=>{
+        let resp =  await fetch('./json/products.json');
```

Comment:
Should we use use Effect hook here?

Reply:
useEffect works as a safeguard in fetching data. Also, it would be helpful in case we make other features or improvements in the future.

Reply:
Thanks. Sorry, now I see you used useEffect.